### PR TITLE
Remote Field & select refactoring

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/tocco-ui/src/EditableValue/typeEditors/RemoteSelect.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Select from 'react-select'
+
+class RemoteSelect extends React.Component {
+  loadOptions = searchTerm => {
+    return this.props.options.fetchOptions(searchTerm).then(res => ({options: res, complete: false}))
+  }
```
Comment (rzueger) [inline packages/tocco-ui/src/EditableValue/typeEditors/RemoteSelect.js]: `{...}` and `return` not needed for this one-liner: `searchTerm => this.props...`

File: packages/tocco-ui/src/EditableValue/typeEditors/MultiSelect.js [hunk 2]
```diff
@@ -1,32 +1,34 @@
 import React from 'react'
 import Select from 'react-select'
+import _isEmtpty from 'lodash/isEmpty'
```
Comment (rzueger) [inline packages/tocco-ui/src/EditableValue/typeEditors/MultiSelect.js]: typo

File: packages/tocco-ui/src/EditableValue/typeEditors/RemoteSelect.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  render() {
+    return (
+      <div>
+        <Select.Async
+          valueKey="key"
+          labelKey="display"
+          loadingPlaceholder="Laden"
+          placeholder=""
+          searchPromptText="Tippen um zu suchen"
```
Comment (rzueger) [inline packages/tocco-ui/src/EditableValue/typeEditors/RemoteSelect.js]: - Tippen, um zu suchen
- Shouldn't be hardcoded here

Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/typeEditors/RemoteSelect.js]: i know. that should be a followup task that can be done by robin or you. or me

**Pull request merged**